src/seedu/addressbook/association/Tagging.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.association;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * Association class between Person and Tags to represent tags that are added or
+ * deleted during a certain session
+ * 
+ * @author varungupta
+ *
+ */
+public class Tagging {
+	private enum Type {
+		ADDED, DELETED
+	};
+
+	public final Type type;
```

Comment:
Note that the our indentation style uses 4 spaces instead of tabs, See here [http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs](url)